x/oracle/keeper/keeper.go

```diff
@@ -67,25 +67,27 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 //-----------------------------------
 // ExchangeRate logic
 
-func (k Keeper) GetBaseExchangeRate(ctx sdk.Context, denom string) (sdk.Dec, error) {
+func (k Keeper) GetBaseExchangeRate(ctx sdk.Context, denom string) (sdk.Dec, sdk.Int, error) {
 	if denom == utils.MicroBaseDenom {
-		return sdk.OneDec(), nil
+		return sdk.OneDec(), sdk.ZeroInt(), nil
```

Comment:
it might be easier for this to be set as the latest period-end block so that callers can handle base vs. non-base denom exchange rates consistently. Something like `ctx.BlockHeight() / votePeriod * votePeriod`. (would also make the eventual switch of base denom to `sei` easier)